packages/frontend/.storybook/main.cjs

```diff
@@ -4,6 +4,7 @@ module.exports = {
     '@storybook/addon-links',
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
+    '@storybook/addon-styling',
```

Comment:
[This is deprecated](https://www.npmjs.com/package/@storybook/addon-styling), we should be using [@storybook/addon-themes](https://github.com/storybookjs/storybook/tree/next/code/addons/themes)